### DISCUSSION TITLE
Background Image not displaying on GitHub pages

### DISCUSSION THREAD

**Discussion by Yash-Kale0101** [community/community#116634] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

I'm new on this platform and I'm facing an issue that is, I've used background tag in my css file in which I've given path of an Image which is not displaying on pages . I tried everything cleared cache, rechecked path, added './' , removed './'. Nothing worked 😕 what should I do please help

This is my repository
https://github.com/Yash-Kale0101/PORTFOLIO.git 
( The css file is in iPortfolio/assets/css/style.css

You will  find the background property in hero section )

**Comment by shahxad-hussain** [comment, marked as answer]
"Hello, I checked, your repository or image link was incorrect, and if you replace this code, it will work 
#hero {
  width: 100%;
  height: 100vh;
  background: url('../img/herobg.jpg') top center;
  background-size: cover;
}

@media screen and (min-width: 768px) {
  #hero {
    background-size: contain;
  }
}

**Reply by Yash-Kale0101** [reply]
Thanks bro it worked
